socorro/app/for_application_defaults.py

```diff
@@ -48,7 +48,10 @@ def find_all_the_apps():
             'reprocess_crashlist':
                 'socorro.external.rabbitmq.reprocess_crashlist.ReprocessCrashlistApp',
             'purge_rmq':
-            'socorro.external.rabbitmq.purge_queue_app.PurgeRabbitMQQueueApp',
+                'socorro.external.rabbitmq.purge_queue_app.PurgeRabbitMQQueueApp',
+            'clear_es_indices': 'socorro.external.es.clear_indices_app.ClearESIndicesApp',
+            'create_recent_es_indices':
+                'socorro.external.es.create_recent_indices_app.CreateRecentESIndicesApp',
```

Comment:
At some point, I think it probably makes more sense to build an Elasticsearch equivalent to the Postgres `setupdb_app` which does all the manipulation rather than have it spread across a few different apps. I wouldn't change anything in this PR, though.